historysync.go

```diff
@@ -212,16 +270,75 @@ func (portal *Portal) sendBackfill(backfillID string, messages []*imessage.Messa
 		portal.bridge.IM.SendBackfillResult(portal.GUID, backfillID, success, idMap)
 	}()
 	batchSending := portal.bridge.SpecVersions.Supports(mautrix.BeeperFeatureBatchSending)
+
 	events, metas, metaIndexes, isRead, err := portal.convertBackfill(messages)
 	if err != nil {
 		portal.log.Errorfln("Failed to convert messages for backfill: %v", err)
 		return false
 	}
-	portal.log.Debugfln("Converted %d messages into %d events to backfill", len(messages), len(events))
+	portal.log.Debugfln("Converted %d messages into %d message events to backfill", len(messages), len(events))
```

Comment:
did you mean to paste lines 314-334 over here? I'm not sure I understand why it was duplicated :X

Reply:
Yes, I'm reusing and reassigning the variable names when calling convertTapbacks. If this isn't best practice, I'm open to changing it. Very new to Go still.